google/datalab/contrib/pipeline/airflow/airflow_vm_startup.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+LOCAL_DAG_PATH=airflow/dags
```

Comment:
You should probably make these explicitly root relative. So /airflow/dags (leading slash).